packages/suite/src/support/messages.ts

```diff
@@ -3833,35 +3833,6 @@ export default defineMessages({
         id: 'TR_TOR_REQUEST_ENABLE_FOR_COIN_JOIN_LEAVE',
         defaultMessage: 'Leave',
     },
-    TR_COINJOIN_ACCESS_ACCOUNT_TITLE: {
-        id: 'TR_COINJOIN_ACCESS_ACCOUNT_TITLE',
-        defaultMessage: 'Access Coinjoin Account',
-    },
-    TR_COINJOIN_ACCESS_ACCOUNT_SUBTITLE: {
-        id: 'TR_COINJOIN_ACCESS_ACCOUNT_SUBTITLE',
-        defaultMessage: 'Privacy enhanced Bitcoin account',
-    },
-    TR_COINJOIN_ACCESS_ACCOUNT_STEP_INITIAL_TITLE: {
-        id: 'TR_COINJOIN_ACCESS_ACCOUNT_STEP_INITIAL_TITLE',
-        defaultMessage: 'Anonymize your funds',
-    },
-    TR_COINJOIN_ACCESS_ACCOUNT_STEP_INITIAL_DESCRIPTION: {
-        id: 'TR_COINJOIN_ACCESS_ACCOUNT_STEP_INITIAL_DESCRIPTION',
-        defaultMessage:
-            'Inputs from multiple senders are combined into a single transaction, making it difficult for external parties to analyze your connection to an old address.',
-    },
-    TR_COINJOIN_ACCESS_ACCOUNT_STEP_INITIAL_FEE_MESSAGE: {
-        id: 'TR_COINJOIN_ACCESS_ACCOUNT_STEP_INITIAL_FEE_MESSAGE',
-        defaultMessage: 'A fee of {fee}% of the used amount is applied',
-    },
```

Comment:
It looks like those messages are still used in component `packages/suite/src/views/wallet/transactions/components/CoinjoinAccountEmpty.tsx`.

Reply:
thanks for noticing, I was resolving the conflicts before merging empty account page...

Reply:
forcepushed, keeping the messages used on CoinjoinAccountEmpty